resource-managers/mesos/src/main/scala/org/apache/spark/deploy/rest/mesos/MesosRestServer.scala

```diff
@@ -153,6 +156,20 @@ private[mesos] class MesosSubmitRequestServlet(
         handleError(s"Received message of unexpected type ${unexpected.messageType}.")
     }
   }
+
+  private[mesos] def validateLabelFormat(properties: Map[String, String]
+                                       , propertyNames: String*): Unit = {
+    propertyNames.foreach { name =>
+      properties.get(name) foreach { label =>
+        try {
+          MesosProtoUtils.mesosLabels(label)
+        } catch {
+          case _ => throw new SubmitRestProtocolException("Malformed label in " +
+            f"${name}: ${label}")
```

Comment:
small nit but string interpolation in Scala is usually done w/ `s"${variable}"` while `f"` is usually done w/ types and digits

Reply:
changed as suggested